build/.pa11yci.json

```diff
@@ -0,0 +1,11 @@
+{
+  "standard": "WCAG2AA",
+  "level": "error",
+  "concurrency": 4,
+  "defaults": {
+    "runners": [
+      "axe"
+    ],
```

Comment:
worth adding `"ignore": [ "notice", "warning" ]` here and seeing if that stops the various "best practice" issues we're now seeing flagged up by axe